code/modules/projectiles/ammunition/ego_ammunition/waw.dm

```diff
@@ -38,6 +43,14 @@
 	desc = "An executive casing."
 	projectile_type = /obj/projectile/ego_bullet/ego_executive
 
+//The only justice scaling gun is a shitty 10 pale pistol
+/obj/item/ammo_casing/caseless/ego_executive/ready_proj(atom/target, mob/living/user, quiet, zone_override = "", atom/fired_from)
+	..()
+	if(isgun(fired_from) && ishuman(user))
+		var/userjust = (get_attribute_level(user, JUSTICE_ATTRIBUTE))
+		var/justicemod = 1 + userjust/100
```

Comment:
Wouldn't it make the gun too strong? Consider updating the bullet's damage.

Reply:
At 100 justice you do 20 pistol damage. That's 40 damage when firing both (same as hornet). It maxes out at around 23 pistol damage at max justice. And at that point you might as well use an aleph weapon.